catalog/src/main/java/org/killbill/billing/catalog/DefaultPlan.java

```diff
@@ -98,12 +98,10 @@
 
     // For deserialization
```

Comment:
⚠️ I think we need the no-arg constructor for off-heap and Redis caching (hence the comment). See 4dc99affdc

Reply:
Confused, the no-arg CTOR exists: `https://github.com/killbill/killbill/blob/work-for-release-0.21-catalog-exp/catalog/src/main/java/org/killbill/billing/catalog/DefaultPlan.java#L100`